### PR TITLE
Rest of V5 Updates

### PR DESCRIPTION
**Pull request by ec2**
**Summary of changes**
Changes introduced in this pull request:
- 

This PR includes the following:
- Updates to the Actor Interface for v5
- All remaining FIPS that are not 0008 or 0013

Here is the specs-actor PR for changes in
- (FIP12) `vm/actor/src/builtin/verifreg/mod.rs`: https://github.com/filecoin-project/specs-actors/pull/1390 
- (Sector Termination Clean Up) `vm/actor/src/builtin/market/`: https://github.com/filecoin-project/specs-actors/pull/1417
- All other changes are "chores", similar to this PR: https://github.com/ChainSafe/forest/pull/1041

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

**Comment by cryptoquick** [comment]
Seeing really weird behavior with the VM taking up all the RAM and then just crawling through the snapshot import. After restarting the VM it looks a little better on the memory usage, but it's still crawling along. The current estimate is that this will take 5 days to import (7300 minutes, to be exact). I even `rm -rf`ed the db dir and copied the snapshot to the local VM SSD in the hopes of improving performance a bit. Not sure if this is consistent with your observations, @ec2 ?

Before:
Screenshot from 2021-08-30 13-41-33
Screenshot from 2021-08-30 13-42-10

Lots of zombie processes maybe?

After reboot, rm -rf, and cp snapshot to home:
Screenshot from 2021-08-30 13-47-51
Screenshot from 2021-08-30 13-48-01

Not sure what's going on. Any changes in this PR that you think might've caused this behavior? I'll try this on my local PC, also.

**Comment by cryptoquick** [comment]
@ec2 Can you link to a lotus git diff I can compare this against?

**Comment by cryptoquick** [comment]
Also, you failed the linter :/

**Review by cryptoquick — CHANGES_REQUESTED** [review]
Everything looks consistent with the go specs actors changes, but I added a couple documentation updates you missed in a PR into this branch:
https://github.com/ChainSafe/forest/pull/1224

Take a look, feel free to merge if it's helpful.

**Review by cryptoquick — APPROVED** [review]
ty for merging muh PR
should be g2g from my end

**Review by ghost — APPROVED** [review]
Great! Thanks @ec2

File: utils/bitfield/src/rleplus/mod.rs [hunk 1]
```diff
@@ -71,12 +71,24 @@ use super::{BitField, Result};
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::borrow::Cow;
 
+// MaxEncodedSize is the maximum encoded size of a bitfield. When expanded into
+// a slice of runs, a bitfield of this size should not exceed 2MiB of memory.
```
[the remark this answers is not in the document]
Comment (cryptoquick) [reply utils/bitfield/src/rleplus/mod.rs:75]: @olibearo I believe it says `runs`, which I think makes sense in this context

[the remark this answers is not in the document]
Comment (cryptoquick) [reply utils/bitfield/src/rleplus/mod.rs:75]: (assuming RLE stands for Run-Length Encoding)

**Pull request merged**